File: sdk/typescript/evaluation.ts
export class EvaluationClient {
  constructor(apiKey) {
    this.apiKey = apiKey;
  }
  runBenchmark(modelId, benchmarkName) {
    return fetch(`https://api.astrovox.ai/evaluation/benchmarks/${benchmarkName}`, {
      method: "POST",
      headers: { Authorization: `Bearer ${this.apiKey}`, "Content-Type": "application/json" },
      body: JSON.stringify({ model_id: modelId }),
    });
  }
}

File: sdk/go/evaluation.go
package evaluation

type Client struct {
	apiKey string
}

func NewClient(apiKey string) *Client {
	return &Client{apiKey: apiKey}
}

func (c *Client) RunBenchmark(modelID, benchmarkName string) (map[string]interface{}, error) {
	return nil, nil
}
